stacker/blueprints/base.py

```diff
@@ -9,10 +10,29 @@
     UnresolvedVariables,
     UnresolvedVariable,
 )
+from .types import CFNType
 
 logger = logging.getLogger(__name__)
 
 
+class CFNParameter(object):
+
+    def __init__(self, value):
+        """Wrapper around a value to indicate a CloudFormation Parameter.
+
+        This allows us to filter out non-CloudFormation Parameters from
+        Blueprint variables when we submit the CloudFormation parameters.
+
+        """
+        self.value = value
```

Comment:
Is it worth checking the type of `value` and bailing if it's not what is expected?